docs/sdk-and-tools/sdk-js/sdk-js-cookbook.md

```diff
@@ -207,100 +219,94 @@ In order to wait for multiple transactions:
 
-transactions.push(new Transaction({
+const tx1 = factory.createESDTTransfer({
+    tokenTransfer: transfer1,
     nonce: 7,
-    sender: new Address("erd1..."),
-    receiver: new Address("erd1..."),
-    data: data,
-    gasLimit: 50000 + 1500 * data.length() + 300000,
+    sender: addressOfAlice,
+    receiver: addressOfBob,
     chainID: "D"
-}));
+});
 ```
 
 [comment]: # (mx-context-auto)
 
 ### Single NFT transfer
 
 ```
-let payment = TokenPayment.nonFungible("SDKJS-38f249", 1);
-let payload = new ESDTNFTTransferPayloadBuilder()
-    .setPayment(payment)
-    .setDestination(new Address("erd1..."))
-    .build();
+const transfer2 = TokenTransfer.nonFungible("TEST-38f249", 1);
 
-transactions.push(new Transaction({
-    nonce: 7,
-    sender: new Address("erd1..."),
-    // Same as sender address!
-    receiver: new Address("erd1..."),
-    data: data,
-    gasLimit: 50000 + 1500 * data.length() + 1000000,
+const tx2 = factory.createESDTNFTTransfer({
+    tokenTransfer: transfer2,
+    nonce: 8,
+    sender: addressOfAlice,
+    destination: addressOfBob,
     chainID: "D"
-}));
+});
 ```
 
 [comment]: # (mx-context-auto)
 
 ### Single SFT transfer
 
 ```
-let payment = TokenPayment.semiFungible("SEMI-9efd0f", 1, 5);
-let payload = new ESDTNFTTransferPayloadBuilder()
-    .setPayment(payment)
-    .setDestination(new Address("erd1..."))
-    .build();
+const transfer3 = TokenTransfer.semiFungible("SEMI-9efd0f", 1, 5);
 
-transactions.push(new Transaction({
-    nonce: 7,
-    sender: new Address("erd1..."),
-    // Same as sender address!
-    receiver: new Address("erd1..."),
-    data: data,
-    gasLimit: 50000 + 1500 * data.length() + 1000000,
+const tx3 = factory.createESDTNFTTransfer({
+    tokenTransfer: transfer3,
+    nonce: 9,
+    sender: addressOfAlice,
+    destination: addressOfBob,
     chainID: "D"
-}));
+});
 ```
 
 [comment]: # (mx-context-auto)
 
 ### Multi ESDT / NFT transfer
 
 ```
-let paymentOne = TokenPayment.nonFungible("SDKJS-38f249", 1);
-let paymentTwo = TokenPayment.fungibleFromAmount("BAR-c80d29", "10.00", 18);
-let payments = [paymentOne, paymentTwo];
-let payload = new MultiESDTNFTTransferPayloadBuilder()
-    .setPayments(payments)
-    .setDestination(new Address("erd1..."))
-    .build();
+const transfers = [transfer1, transfer2, transfer3];
 
-transactions.push(new Transaction({
-    nonce: 7,
-    sender: new Address("erd1..."),
-    // Same as sender address!
-    receiver: new Address("erd1..."),
-    data: data,
-    gasLimit: 50000 + 1500 * data.length() + 1000000 * payments.length,
+const tx4 = factory.createMultiESDTNFTTransfer({
+    tokenTransfers: transfers,
+    nonce: 10,
+    sender: addressOfAlice,
+    destination: addressOfBob,
     chainID: "D"
-}));
+});
 ```
 
+<!-- END_INCLUDE_FILE -->
+
+<!-- BEGIN_INCLUDE_FILE { "url": "https://raw.githubusercontent.com/multiversx/mx-sdk-js-examples/next/cookbook/generated/contracts_01_deployments.md" } -->
```

Comment:
might reference a commit hash instead of the branch name? otherwise, make sure the `next` branch isn't removed in that repo

Reply:
Added commit hash.